scanner/Makefile

```diff
@@ -0,0 +1,152 @@
+# Store tooling in a location that does not affect the system.
+GOBIN := $(CURDIR)/.gobin
+export GOBIN
+PATH := $(GOBIN):"$(PATH)"
+export PATH
+
+# Set to empty string to echo some command lines which are hidden by default.
+SILENT ?= @
+
+SHELL = env GOBIN=$(GOBIN) PATH=$(PATH) /bin/bash
```

Comment:
You already exported GOBIN and PATH above, so this is redundant. Or did you forget `-i`?

Reply:
artifact from old Scanner repo. Removed env vars